pf-freebsd-setup/azuredeploy.json

```diff
@@ -0,0 +1,342 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "frontendPrivateNicIP1": {
+      "type": "string",
+      "defaultValue": "10.0.2.5",
+      "metadata": {
+        "description": "Frontend VM private IP address which resides in private subnet."
+      }
+    },
+    "frontendPrivateNicIP2": {
+      "type": "string",
+      "defaultValue": "10.0.0.5",
+      "metadata": {
+        "description": "Frontend VM public IP address which resides in public subnet."
+      }
+    },
+    "backendVM1PrivateNicIP": {
+      "type": "string",
+      "defaultValue": "10.0.2.6",
+      "metadata": {
+        "description": "Backend VM1 private IP address which resides in private subnet."
+      }
+    },
+    "backendVM2PrivateNicIP": {
+      "type": "string",
+      "defaultValue": "10.0.2.7",
+      "metadata": {
+        "description": "Backend VM2 private IP address which resides in private subnet."
+      }
+    },
+    "frontendVmSize": {
+      "type": "string",
+      "defaultValue": "Standard_A4",
+      "metadata": {
```

Comment:
Equal or more than 2 NICs meets the requirement. I will try to add all allowed values.